tutorial.md

```diff
@@ -316,11 +316,115 @@ Shops require several columns:
   - sell: The amount of money given for selling this to the shop 
 
 # Scripting
-We use Pyk to implement our scripts. For a tutorial on Pyk syntax, see [here](https://github.com/IAmTomahawkx/pyk#syntax)
+We use Pyk to implement our scripts. 
+For some help with Pyk syntax  >>>>>>>>> [CLICK HERE](https://github.com/IAmTomahawkx/pyk#syntax)
+
+Some Basics:
+
+# How to create a script
+  - Write your code and save it in a .txt file
+  - Upload the .txt file to your discord server, with the message !script create scriptname
```

Comment:
might be good to use the default prefix "rp!" instead so that people dont ask why its not working